### DISCUSSION TITLE
Is is possible to use Markdown inside HTML blocks?

### DISCUSSION THREAD

**Discussion by matthiasprieth** [datenstrom/community#348] [Ask a question] [answered]
Hey guys! Really like Yellow so far! Great simple CMS!

Is it possible to have self-written plugins within HTML-Code in a `/yellow/content/example.txt` file? It only works if i place my plugins after a closed HTML-element.

/yellow/content/example.txt
```
<p>
[myplugin]
</p>
```

renders to
```
[myplugin]
```

instead of
```
result returned in my plugin
```

**Comment by schulle4u** [comment]
Plugin shortcuts within HTML blocks are currently not possible. You however can put any surrounding HTML into your plugin, it will then become part of the plugin shortcut and give the same result. Do you have any special example that would require shortcuts within HTML elements? 

Steffen

**Comment by schulle4u** [comment, marked as answer]
Thanks to a friendly reminder behind the scenes, markdown inside HTML [actually is possible](https://michelf.ca/projects/php-markdown/extra/#markdown-attr), thus you should be able to include your plugin like this. 

```
<p markdown="1">
[myplugin]
</p>
```

Steffen

**Comment by matthiasprieth** [comment]
Thanks for the quick response! In my case the content creators needed it for having dynamic numbers in the content sometimes in a `<ul><li>` list, sometimes in a paragraph & so on. So I would have needed to create several plugins or one plugin with the same logic, but different templates.

The solution described above works & is better for me, as it is more flexible. Thanks!
